internal/controller/nodedisruption_controller.go

```diff
@@ -216,6 +211,7 @@ func (ndr *NodeDisruptionResolver) ValidateDisruption(ctx context.Context, budge
 				Reason:    "No more disruption allowed",
 				Ok:        false,
 			})
+			fmt.Println("Stopping !!!!!!!!!!!!")
```

Comment:
I don't know if this trace is kept voluntarily :)

Reply:
Oups